test/functional/cmdline_options_tester/build.xml

```diff
@@ -45,11 +45,10 @@
 
 	<target name="compile" depends="init" description="compile the source ">
 		<echo>Ant version is ${ant.version}</echo>
-		<echo>============COMPILER SETTINGS============</echo>
```

Comment:
Why was this line removed?

Reply:
Missed this deletion in my local diff - fixing!